cloudformation/functions/get_groups_from_policy.py

```diff
@@ -0,0 +1,99 @@
+from json import loads
+from json.decoder import JSONDecodeError
+
+
+UNGLOBBABLE_OPERATORS = ("StringEquals", "ForAnyValue:StringEquals")
+UNSUPPORTED_OPERATORS = (
+    "StringNotEquals", "ForAnyValue:StringNotEquals",
+    "StringNotLike", "ForAnyValue:StringNotLike"
+)
+VALID_OPERATORS = (
+    "StringEquals", "ForAnyValue:StringEquals",
+    "StringLike", "ForAnyValue:StringLike",
+)
+
+VALID_AMRS = (
```

Comment:
would be nice if L15 to L22 were in a config file, though, not required for MVP. The goal being "anyone can use this"

Reply:
I think that sounds reasonable, but can we have that be a separate PR instead of getting it into this one?  :)